base/doc/clsguide.tex

```diff
@@ -977,6 +977,30 @@ \subsection{Passing options around}
 handler, but this handler is not used for |landscape| because that option is
 explicitly declared.
 
+\subsection{Testing dates of other code}
+
+\begin{decl}
+  |\IfPackageAtLeast(TF)| \arg{package} \arg{date} \arg{true code} \arg{false code}\\
+  |\IfClassAtLeast(TF)| \arg{class} \arg{date} \arg{true code} \arg{false code}\\
+  |\IfFileAtLeast(TF)| \arg{file} \arg{date} \arg{true code} \arg{false code}
```

Comment:
I think these parentheses can be dropped.
- Existence of no-parentheses forms (at least one occurrence for each command) in PDF help when searching.
- `\arg{true code} \arg{false code}` corresponds to `TF` only.

Suppose there will be more `clsguide` subsections describing `\If...TF` conditionals, a in-detail syntax and description of the single-branch forms could be added at the first of those subsections, for the first command, which then can be referred to in following sections.

```suggestion
  |\IfPackageAtLeastTF| \arg{package} \arg{date} \arg{true code} \arg{false code}\\
  |\IfClassAtLeastTF| \arg{class} \arg{date} \arg{true code} \arg{false code}\\
  |\IfFileAtLeastTF| \arg{file} \arg{date} \arg{true code} \arg{false code}
```

Reply:
> We use the `(TF)` convention in `expl3` and I felt it was clearer - but let's see what others feel.
> <sub>_Originally posted by @josephwright in https://github.com/latex3/latex2e/pull/1452#discussion_r1738892279_</sub>

But in syntax blocks in `interface3.pdf`, there're no `\func...:...(TF) ... {<true code>} {<false code>}`, just `\func...:...TF ... {<true code>} {<false code>}`.

(So sorry I posted reviews as separate comments, then deleted the comments and reposted them as part of a review, _again_.)

Reply:
True. Though in `interface3.pdf`, even the marginal versions don't print _explicit_ parentheses enclosing `TF`. We need a better way to document `TF` forms in `clsguide`.

![image](https://github.com/user-attachments/assets/578564c5-128c-4219-841e-9058db13aeb9)

![image](https://github.com/user-attachments/assets/bab8561c-c9da-4d7e-aa4d-59b7fef20850)

Reply:
Maybe just use \If...TF and have a convention section up front that has a subsection on conditionals mentioning that "most"  (I'm reculant to say all really) TF commands also have variants with T or F for cases where the other branch is unnecessary.